spec/spec_helper.rb

```diff
@@ -30,6 +30,11 @@
   # instead of true.
   # config.use_transactional_fixtures = true
   config.before(:each) do
+    if defined?(Rails)
+      @original_rails = Rails
+      Object.send(:remove_const, :Rails)
+    end
```

Comment:
I have added this so that the test suit does not affect each other's **Rails** const. When we run **XapianDb::Railtie** we initializes a global Rails const in the same test process. As a result methods like [defined?(Rails)](https://github.com/gernotkogler/xapian_db/blob/master/lib/xapian_db/database.rb#L112) become true and fail other tests with the following error.

> Failure/Error: Rails.logger.debug "XapianDb search (#{(Time.now - start) * 1000}ms) #{expression}" if defined?(Rails)
     NoMethodError:
       undefined method debug for nil:NilClass
     # ./lib/xapian_db/database.rb:117